birdhouse/default.env

```diff
@@ -84,19 +84,14 @@ export SERVER_LICENSE_URL='${__DEFAULT__SERVER_LICENSE_URL}'
 # Those will not be set explicitly as defaults to ensure they are overridden explicitly by the instance.
 # These values would be detected only if the instance was configured using a copy of 'env.local.example'.
 export __DEFAULT__MAGPIE_SECRET=itzaseekrit
-#export __DEFAULT__MAGPIE_ADMIN_USERNAME=admin
```

Comment:
Should this one remain to warn about it not being modified?

Reply:
Yeah I was debating whether to keep them or remove them....

I guess I was thinking that it's OK if the username stays as "admin" as long as we recommend they change the password.

But in the end, we're just warning the user that we recommend that it *should* be changed from the default. If they want to keep it, they can.

I guess I can put them back in